app/Http/Controllers/Wiki/LanguagesController.php

```diff
@@ -86,11 +86,14 @@ public function index()
         $code                  = checkParam('code|iso|language_code');
         $include_translations  = checkParam('include_translations|include_alt_names');
         $name                  = checkParam('name|language_name');
+        $show_bibles           = checkBoolean('show_bibles');
         $limit                 = (int) (checkParam('limit') ?? 50);
         $page                  = checkParam('page') ?? 1;
 
         // remove pagination for bibleis and gideons (temporal fix)
         list($limit, $is_bibleis_gideons) = forceBibleisGideonsPagination($this->key, $limit);
+        // this is a band aid to reduce the query with a forced asset_id if uses show_bibles
```

Comment:
add a note: this can be removed when bibleis and gideons no longer require a non-paginated response